app/controllers/project.py

```diff
@@ -204,17 +204,51 @@ def get(self):
         page = request.args.get('page', 1, type=int)
         per_page = request.args.get('per_page', 10, type=int)
         keywords = request.args.get('keywords', '')
+        disabled = request.args.get('disabled')
+        project_type = request.args.get('project_type')
+        organisation = request.args.get('organisation')
+
 
         project_schema = ProjectSchema(many=True)
         projects = []
+        pagination_data = {}
+
+        filter_mapping = {
+        'disabled': disabled,
+        'project_type': project_type,
+        'organisation': organisation
+        }
+
+        # count items per project category 
+        project_metadata = {}
+        for category in filter_mapping.keys():
+            distinct_counts = (Project.query.with_entities(getattr(Project, category), func.count(getattr(Project, category))).group_by(getattr(Project, category)).all())
+            project_metadata[category] = {key: value for key, value in distinct_counts}
```

Comment:
Out of curiosity, does the frontend really have use for all these organisations that are computed each time this endpoint is hit?
<img width="347" alt="Screenshot 2023-08-18 at 1 25 40 AM" src="https://github.com/crane-cloud/backend/assets/18049550/1f1874a0-f809-4407-a5b0-26fcb7029062">

Reply:
This slows down the endpoint, thats for sure. Maybe you create a different route to return these things

Reply:
@Mubangizi I have removed the organisation filter